client/src/components/CardDetail.js

```diff
@@ -0,0 +1,493 @@
+import React, { Fragment, useState } from "react";
```

Comment:
There are a couple of errors on the console, can you please resolve them before merging